client/js/screens/Signup/SignupContainer.js

```diff
@@ -0,0 +1,10 @@
+import React, {Component} from 'react';
```

Comment:
this isn't doing much...can we just have the Signup component?

Reply:
We can. We were told that it's best practice to build screens with containers incase you need to add logic in the future.